django/core/management/commands/dumpdata.py

```diff
@@ -144,7 +144,16 @@ def get_objects(count_only=False):
             Collate the objects to be serialized. If count_only is True, just
             count the number of objects to be serialized.
             """
-            models = serializers.sort_dependencies(app_list.items())
+            if use_natural_foreign_keys:
+                models = serializers.sort_dependencies(app_list.items(), allow_loops=True)
```

Comment:
If I understand correctly _cycles_ might be a better term than _loops_ here.

Reply:
Agreed, I'll change that.

Reply:
Done, will push soon.